pywinauto/controls/common_controls.py

```diff
@@ -3286,14 +3286,109 @@ class IPAddressWrapper(HwndWrapper.HwndWrapper):
         controltypes = [IUIA().UIA_dll.UIA_CalendarControlTypeId]
     has_title = False
+    #----------------------------------------------------------------
+    def __init__(self, hwnd):
+        "Initialise the instance"
+        #HwndWrapper.HwndWrapper.__init__(self, hwnd)
+        super(CalendarWrapper, self).__init__(hwnd)
+
+    #----------------------------------------------------------------
+    def get_current_date(self):
+        "Get the currently selected date"
+        
+        remote_mem = RemoteMemoryBlock(self)
+        system_date = win32structures.SYSTEMTIME()
+        remote_mem.Write(system_date)
+        
+        res = self.send_message(win32defines.MCM_GETCURSEL , 0, remote_mem)
+        remote_mem.Read(system_date)
+        del remote_mem
+        
+        if res == 0:
+            raise RuntimeError('Failed to get time in Calendar')
+        
+        #year = system_time.wYear
+        #month = system_time.wMonth
+        #day_of_week = system_time.wDayOfWeek
+        #day = system_time.wDay
+        #hour = system_time.wHour
+        #minute = system_time.wMinute
+        #second = system_time.wSecond
+        #milliseconds = system_time.wMilliseconds
+        #return (year, month, day_of_week, day, hour, minute, second, milliseconds)
+        return system_date
+    # Non PEP-8 alias
+    GetCurrentDate = get_current_date
 
+    #----------------------------------------------------------------
+    def set_date(self, year, month, day_of_week, day):
+        "Set the currently selected date"
+        
+        remote_mem = RemoteMemoryBlock(self)
+        system_time = win32structures.SYSTEMTIME()
+        
+        system_time.wYear = year
+        system_time.wMonth = month
+        system_time.wDayOfWeek = day_of_week
+        system_time.wDay = day
+        system_time.wHour = 0
+        system_time.wMinute = 0
+        system_time.wSecond = 0
+        system_time.wMilliseconds = 0
+        
+        remote_mem.Write(system_time)
+        
+        res = self.send_message(win32defines.MCM_SETCURSEL, win32defines.GDT_VALID, remote_mem)
+        del remote_mem
+        
+        if res == 0:
+            raise RuntimeError('Failed to set time in Calendar')
+    # Non PEP-8 alias
+    SetDate = set_date
+
+    #----------------------------------------------------------------
+    def get_calendar_border(self):
+        "Get the calendar border"
+        return self.send_message(win32defines.MCM_GETCALENDARBORDER, 0,0)
+    # Non PEP-8 alias
+    GetCalendarBorderWidth = get_calendar_border
+
+    #----------------------------------------------------------------
+    def set_calendar_border(self, border):
+        "Set the calendar border"
+        self.send_message(win32defines.MCM_SETCALENDARBORDER, True, border)       
+    # Non PEP-8 alias
+    SetCalendarBorderWidth = set_calendar_border
+
+    #----------------------------------------------------------------
+    def get_calendars_cout(self):

```

Comment:
Typo: must be `count()` (no need to add `get_` because there are no `set_` method).